.agentkit/templates/claude/hooks/stop-build-check.ps1

```
@@ -72,10 +72,26 @@ function Send-Block {
 # -- Auto-detect stack and run checks --------------------------------------
 $ranCheck = $false
 
+# Compute changed files once (used for all per-language gates below). Only
+# inspect files changed since HEAD (staged + unstaged). This keeps the hook
+# essentially free when only docs or unrelated configuration were touched.
+$changedFiles = @()
+if ((Get-Command "git" -ErrorAction SilentlyContinue) -and
+    (& git -C $cwd rev-parse --is-inside-work-tree 2>$null) -eq "true") {
```

PSScriptAnalyzer (1.25.0)

[warning] Did you mean to use the redirection operator '>'? The comparison operators in PowerShell are '-gt' (greater than) or '-ge' (greater or equal).

(PSPossibleIncorrectUsageOfRedirectionOperator)